packages/console/src/components/ItemPreview/index.tsx

```diff
@@ -31,6 +32,7 @@ function ItemPreview({ title, subtitle, icon, to, size = 'default' }: Props) {
         )}
         {!to && <div className={styles.title}>{title}</div>}
         {subtitle && <div className={styles.subtitle}>{subtitle}</div>}
+        {suffix}
```

Comment:
Using a "tag" props with i18n keys maybe not be a good solution since tags will have different styles and maybe the suffix content is not a tag.

Reply:
1. i suppose tag should have an aligned style in the `ItemPreview` component
2. title and subtitle can be arbitrary component as well. then the meaning of a prop is trivial, since every prop can be a component